raster/src/main/scala/geotrellis/raster/Tile.scala

```diff
@@ -292,4 +292,10 @@ trait Tile extends CellGrid with IterableTile with MappableTile[Tile] {
     }
     s
   }
+
+  def isNoDataTile = {
+    var empty = true
+    foreach { z => if (isData(z)) empty = false }
```

Comment:
Have you an idea how to call it? I like the `isEmpty`, but not sure that it is a correct function name and can be confusing.